frontend/src/metabase/actions/containers/ActionCreator/FormCreator/FormFieldEditor/FormFieldEditor.tsx

```diff
@@ -97,6 +98,14 @@ function FormFieldEditor({
     });
   };
 
+  const handleVisibilityChange = ({ hidden }: { hidden: boolean }) => {
+    setHidden(hidden);
```

Comment:
It seems we can (and should) remove the local state

Reply:
you're right, it's much easier